packages/devtools-reps/src/shared/grip-length-bubble.js

```diff
@@ -0,0 +1,43 @@
+const PropTypes = require("prop-types");
+
+const { wrapRender } = require("../reps/rep-utils");
+const { MODE } = require("../reps/constants");
+const { ModePropType } = require("../reps/array");
+
+const dom = require("react-dom-factories");
+const { span } = dom;
+
+GripLengthBubble.propTypes = {
+  object: PropTypes.object.isRequired,
+  maxLengthMap: PropTypes.instanceOf(Map).isRequired,
+  getLength: PropTypes.func.isRequired,
+  mode: ModePropType,
+  visibilityThreshold: PropTypes.number
+};
+
+function GripLengthBubble(props) {
+  const {
+    object,
+    mode = MODE.SHORT,
+    visibilityThreshold = 5,
```

Comment:
i think the threshold should be 2, because some objects can be quote lengthy and hard to count (e.g. `new Map([[document, location]])`)